src/lambdas/mps-report/src/generateReport.ts

```diff
@@ -0,0 +1,137 @@
+import getCourtErrors from "./getCourtErrors"
+import { headers } from "../config/mpsHeadingKeys"
+import getOffenceCode from "./getOffenceCode"
+import AnnotatedHearingOutcome from "./types/AnnotatedHearingOutcome"
+import { PostgresGateway } from "@bichard/postgres-gateway"
+import { isError } from "@bichard/types"
+import { xml2js } from "xml-js"
+import { getText } from "./types/XmlStringCell"
+import getResultQualifierCodes from "./getResultQualifierCodes"
+import getDurationAndAmount from "./getDurationAndAmount"
+import OffenceDetails, { isMultiple } from "./types/OffenceDetails"
+import OffenceResult from "./types/OffenceResult"
+import HearingOutcomeCase from "./types/HearingOutcomeCase"
+
+interface ReportRowResultQuery {
+    court_date: string
+    court_code: string
+    force_code: string
+    is_urgent: string
+    defendant_name: string
+    triggers: string
+    error_report: string
+}
+
+function UpdateCommonHeaders(row: ReportRowResultQuery, hearingOutcomeCase: HearingOutcomeCase, offence:OffenceDetails) {
+  return [
+    row.court_date, // Hearing Date
+    row.court_code, // Court
+    row.force_code, // Force
+    row.is_urgent, // Urgent
+    row.defendant_name, // Defendant
+    getText(hearingOutcomeCase.HearingDefendant.DefendantDetail?.BirthDate), // DoB
+    getText(hearingOutcomeCase.HearingDefendant.ArrestSummonsNumber), // ASN
+    getText(hearingOutcomeCase.PTIURN), // URN
+    row.triggers, // Triggers
+    row.error_report, // Errors
+    getOffenceCode(offence.CriminalProsecutionReference), // Offence Code
+    getText(offence.ActualOffenceStartDate.StartDate), // Offence Start Date
+    getText(offence.LocationOfOffence), // Offence Location
+  ]
+}
+
+export default async (gateway: PostgresGateway) => {
+  const rows = await getCourtErrors(gateway)
+  if (isError(rows)) {
+    return rows
+  }
+  
+  let result = []
+  result.push(headers.join(","))
+  for (let i = 0; i < rows.length; i = i + 1) {
+    let annotatedMsg: string = rows[i].annotated_msg.replace(/br7:/g, "").replace(/ds:/g, "")
+    let annotatedMsgObject = xml2js(annotatedMsg, { compact: true }) as AnnotatedHearingOutcome
```

Comment:
It seems we don't change the value of `annotatedMsg` and `annotatedMsgObject` in this scope. Can we change `let` to `const`?